CS3219-AY2223S1/cs3219-project-ay2223s1-g42
# fix reset password form bug

PR summary:
* Fix reset password form bug
* Use tan stack useMutation in verify email

Tests:
* Verify that verify email still works
* Verify that password can be reset

Closes #101.
